.github/workflows/wheels.yml

```diff
@@ -140,16 +140,8 @@ jobs:
 
       - name: (Un)comment packages
         run: |
-          requirement_files="requirements_all.txt requirements_diff.txt"
+          requirement_files="requirements_wheels.txt requirements_diff.txt"
           for requirement_file in ${requirement_files}; do
-            sed -i "s|# fritzconnection|fritzconnection|g" ${requirement_file}
```

Comment:
fritzconnection is not used by any integration

Reply:
It's used by fritz and fritzbox_callmonitor.

https://github.com/home-assistant/core/blob/093aee672cb103e30a7dedc5088d29c5f718cafa/homeassistant/components/fritz/manifest.json#L10

https://github.com/home-assistant/core/blob/093aee672cb103e30a7dedc5088d29c5f718cafa/homeassistant/components/fritzbox_callmonitor/manifest.json#L10

Reply:
That one isn't commented though, so yes we can remove it.